src/psycopmlutils/loaders/raw/load_medications.py

```diff
@@ -62,7 +62,7 @@ def load(
                 atc_code matches atc_code_prefix, 0 if not.Defaults to
                 {atc_code_prefix}_value.
             load_prescribed (bool, optional): Whether to load prescriptions. Defaults to
-                True. Beware incomplete until sep 2016.
```

Comment:
@MartinBernstorff are you using this loader? if so, notice this

Reply:
Yes, there's a warning added by me as well ;-)

But good point, will have to fix in the future.